src/test/java/com/wikia/webdriver/testcases/adstests/TestAdFliteSkinPresence.java

```diff
@@ -0,0 +1,84 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+public class TestAdFliteSkinPresence extends TemplateNoFirstLoad {
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteSkin",
+      groups = "TestSkinPresence_GeoEdgeFree"
+  )
+  public void TestAdFliteSkinPresence(String wikiName, String article,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method to prevent any misunderstanding or make it a constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1223)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)